osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -60,6 +60,11 @@ public double Distance
         public List<List<SampleInfo>> RepeatSamples { get; set; } = new List<List<SampleInfo>>();
         public int RepeatCount { get; set; }
 
+        /// <summary>
+        /// The length of one repeat if any repeats are present, otherwise it equals the <see cref="Duration"/>.
+        /// </summary>
+        public double SpanDuration => RepeatCount > 0 ? Distance / Velocity : Duration;

```

Comment:
How about `Duration / this.SpanCount()`? And xmldoc can be `The length of one span of this <see cref="Slider"/>.`